target/linux/ipq40xx/image/Makefile

```diff
@@ -217,6 +217,32 @@ define Device/compex_wpj428
 endef
 TARGET_DEVICES += compex_wpj428
 
+define Device/dlink_dap-2610
+	$(call Device/FitImageLzma)
+	DEVICE_VENDOR := D-Link
+	DEVICE_MODEL := DAP-2610
+	SOC := qcom-ipq4018
+	DEVICE_DTS_CONFIG := config@ap.dk01.1-c1
+	WRGG_DEVNAME := /dev/mtdblock/8
+	WRGG_SIGNATURE := wapac30_dkbs_dap2610
```

Comment:
Bit OT: Are the WRGG_VERSION, WRGG_MODEL and WRGG_BUILDNO set somewhere that I can't see?

(If not, you could just drop them from the Build/wrgg-image for now, if you plan on moving the helper to this file anyway).

Reply:
Ok.